subprojects/docs/src/samples/userguide/dependencyManagement/dependencyLocking/lockingClasspathConfiguration/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+// tag::locking-classpath[]
+buildscript {
+    configurations.getByName("classpath") {
```

Comment:
Because I need a closure allowing to configure the configuration. Your sugested syntax causes the following error:
```
  Line 3:     configurations["classpath"] {
              ^ Expression 'configurations["classpath"]' of type 'Configuration!' cannot be invoked as a function. The function 'invoke()' is not found
```